### PR TITLE
fix: passcode during call - cleanup [task][WPB-7111]

### PR DESCRIPTION
**Pull request by markusfassbender**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-7111" title="WPB-7111" target="_blank">BugWPB-7111</a>  [iOS] Bund builds asks for passcode during call
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

See jira ticket.

Looking at the code I found some simplifications and things that don't need to be have `public` scope.

### Dependencies (Optional)

💡 Merges into feature branch!

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

- Build and test suite

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: wire-ios-data-model/Source/Utilis/EAR/EARKeyRepositoryFailure.swift [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
```
Comment (caldrian) [inline wire-ios-data-model/Source/Utilis/EAR/EARKeyRepositoryFailure.swift]: optional: There is nothing in this file which uses foundation, so the import could be removed.

File: wire-ios-data-model/Source/Utilis/EAR/EARPrivateKeys.swift [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
```
Comment (caldrian) [inline wire-ios-data-model/Source/Utilis/EAR/EARPrivateKeys.swift]: optional: There is nothing in this file which uses foundation, so the import could be removed.

File: wire-ios-data-model/Source/Utilis/EAR/EARPublicKeys.swift [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
```
Comment (caldrian) [inline wire-ios-data-model/Source/Utilis/EAR/EARPublicKeys.swift]: optional: There is nothing in this file which uses foundation, so the import could be removed.

File: wire-ios-data-model/Source/Utilis/EAR/EARServiceDelegate.swift [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
```
Comment (caldrian) [inline wire-ios-data-model/Source/Utilis/EAR/EARServiceDelegate.swift]: optional: There is nothing in this file which uses foundation, so the import could be removed.

File: wire-ios-data-model/Source/Utilis/EAR/EARServiceFailure.swift [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
```
Comment (caldrian) [inline wire-ios-data-model/Source/Utilis/EAR/EARServiceFailure.swift]: optional: There is nothing in this file which uses foundation, so the import could be removed.

**Review by caldrian — APPROVED** [review]
nice work

**Review by typfel — COMMENTED** [review]
Can you point out the actual fix for showing the passcode during a call?

**Comment by markusfassbender** [comment]
As description says this is only a task PR that merges into a feature branch. It's not the actual fix, just things I came along and wanted to improve ;)

**Review by typfel — APPROVED** [review]

**Pull request merged**